backend/app/routes/faq/post.js

```diff
@@ -0,0 +1,20 @@
+const to = require('await-to-js').default;
+const FAQ = require('../../models/faq');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+module.exports = async (req, res, next) => {
+  const [err] = await to(FAQ.create({ ...req.body }));
```

Comment:
you are not taking isActive in req body, take care of that, you need to assign it here i guess

Reply:
I tested it, it is taking isActive as well

Reply:
I mean, in the req body you will not get isActive

Reply:
I'm not sure if I understood you correctly, the screenshots below are of post.js and the response, that I mentioned earlier.

![image](https://user-images.githubusercontent.com/69788174/114761188-cbc39f00-9d7d-11eb-8102-f7a2009604fa.png)

![image](https://user-images.githubusercontent.com/69788174/114760702-3e804a80-9d7d-11eb-9d18-20e791252c09.png)

Reply:
Yes, I understood
You will not get isActive field in the request body always

Reply:
how will it work when request body contains
question answer and tags only

Reply:
Do you mean something like this?
![image](https://user-images.githubusercontent.com/69788174/114762369-4ccf6600-9d7f-11eb-988f-3125380475de.png)

Reply:
I've set it to true by default in case it is not provided in req.body. Would that be alright?